### DISCUSSION TITLE
Any chance for a future „better gfn“?

### DISCUSSION THREAD

**Discussion by Sppedtech** [redphx/better-xcloud#146] [General]
Hi,

First of all thank you for creating this awesome extension/userscript for xcloud.

Is there any way to create a similar script for using GeForce-Now and control the client side settings? The most common features would be to change the user-agents, screen-ratios (16:9 + 16:10) and resolution (like 1080p) on mobile, especially on iOS/iPadOS.

**Comment by redphx** [comment]
I don't use GFN (and don't plan to) so I don't know anything about it.  
If they have a web client then I think it's possible for a "Better GFN" script, but I won't be the one who will make it.

**Reply by Sppedtech** [reply]
ok, that is totally fine and understandable. 
Than I have to take this callange on my own. Do you have any tipps for me how to get proper skills for what you have done with better-xcloud? I am IT professional but no good programming skills. Maybe you have some useful links. 
thanks in advance

**Comment by Julmik647** [comment]
There are some browser extensions for Geforce Now that I found:
[GeForce Now Web Enabler](https://chromewebstore.google.com/detail/geforce-now-web-enabler/ioaplpondpicpdlfoccpaenbdinbdbpi) - play Geforce Now on unsupported devices
[GeForce-Now-Web](https://github.com/bulkypanda/GeForce-Now-Web) - play Geforce Now on unsupported devices
[Show GeForce Now on Steam](https://chromewebstore.google.com/detail/show-geforce-now-on-steam/igigngnkilldoncfnbcknnabjmiggbee?hl=en&authuser=0) - Shows if you can play games in GeForce Now while you are in steam website, not really for Geforce Now ([Reddit post](https://www.reddit.com/r/GeForceNOW/comments/v7dhpz/i_created_a_chrome_extension_for_geforce_now_users/))
[Adblock Plus](https://adblockplus.org/), [Ublock Origin](https://ublockorigin.com/) and other ad blockers - block ads on free tier
And also some Better xCloud features are normally supported by Geforce Now. For example: [Better xCloud allows you to enable 1080p on xCloud](https://better-xcloud.github.io/#key-features) when [Geforce now has 4K support](https://gist.github.com/KhaosT/a4d90479e3cb0aae959ad55b08cd8358#using-geforce-now).
